### DISCUSSION TITLE
Connecting to wallet

### DISCUSSION THREAD

**Discussion by GowthamMSG** [Chia-Network/chia-blockchain#7110] [Support]
I just opened the App & Its showing connecting to wallet.

**Comment by loppefaaret** [comment]
if it is just showing "Connecting To Wallet..." and stays there for several minutes, you likely need to nuke the wallet database with these instructions:
WINDOWS WALLET DATABASE RESET

Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
mv ~\.chia\mainnet\wallet\db\block* ~\Desktop\
```
Then try opening up the Chia app again
if it worked, you can remove the files we put on the Desktop!

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
